README.md

```diff
@@ -1,26 +1,67 @@
-[![Maintained by Gruntwork.io](https://img.shields.io/badge/maintained%20by-gruntwork.io-%235849a6.svg)](https://gruntwork.io/?ref=repo_aws_consul)
+<!--
+:type: service
+:name: HashiCorp Consul
+:icon: /_docs/consul.png
+:description: Deploy a Consul cluster. Supports automatic bootstrapping, DNS, Consul UI, and auto healing.
+:category: Service discovery, service mesh
+:cloud: aws
+:tags: consul, mesh
+:license: gruntwork
+:built-with: terraform, bash
+-->
+
+
+# Consul AWS Module
+
+[![Maintained by Gruntwork.io](https://img.shields.io/badge/maintained%20by-gruntwork.io-%235849a6.svg)](https://gruntwork.io/?ref=repo_terraform_aws_consul)
+![Terraform Version](https://img.shields.io/badge/tf-%3E%3D0.12.0-blue.svg)
+
+This repo contains a Module for running Kubernetes clusters on [AWS](https://aws.amazon.com) using [Elastic Kubernetes Service (EKS)](https://docs.aws.amazon.com/eks/latest/userguide/clusters.html) with [Terraform](https://www.terraform.io).
+
+![Terraform AWS  Consul](https://raw.githubusercontent.com/hashicorp/terraform-aws-consul/master/_docs/architecture.png)
+
+
 # Consul AWS Module
 
-This repo contains a set of modules in the [modules folder](https://github.com/hashicorp/terraform-aws-consul/tree/master/modules) for deploying a [Consul](https://www.consul.io/) cluster on 
-[AWS](https://aws.amazon.com/) using [Terraform](https://www.terraform.io/). Consul is a distributed, highly-available 
-tool that you can use for service discovery and key/value storage. A Consul cluster typically includes a small number
-of server nodes, which are responsible for being part of the [consensus 
-quorum](https://www.consul.io/docs/internals/consensus.html), and a larger number of client nodes, which you typically 
-run alongside your apps:
 
-![Consul architecture](https://github.com/hashicorp/terraform-aws-consul/blob/master/_docs/architecture.png?raw=true)
 
+## Features
+* Secure service communication and observe communication between your services without modifying their code.
+* Automate load balancer.
+* Provides any number of health checks.
+* Multi-Data centers out of the box.
```

Comment:
Replace with:

* Deploy Consul servers and agents 
* Automatic bootstrapping
* Auto healing
* Auto DNS configuration
* Consul UI

Reply:
Okay done